pandas/core/generic.py

```diff
@@ -1154,29 +1154,34 @@ def rename_axis(self, mapper, axis=0, copy=True, inplace=False):
         --------
         **Series**
 
-        >>> s = pd.Series([1, 2, 3])
-        >>> s.rename_axis("foo")
-        foo
-        0    1
-        1    2
-        2    3
-        dtype: int64
+        >>> s = pd.Series(["dog", "cat", "monkey"])
+        >>> s.rename_axis("animal")
```

Comment:
can you show what's the name of the axis before renaming it

Reply:
Can you add this, it was originally requested by @jreback, and it makes sense to have. After this we can merge.

Reply:
Doesn't line 1158 show the contents of `s` including the name of the axis which is not set at the time?

Reply:
I think there is a bug in Git. If you go into `Files changed`, it shows you that line 1158 prints the value of `s`

Reply:
@jreback ?